### PR TITLE
Login screen

### PR DESCRIPTION
**Pull request by JudithBLinder**
arranged few of the styling variables.
some changes to login screen - DABL

this the login screen as it is:
Screen Shot 2021-04-13 at 3 01 51 PM

the white stripes on the right side and the bottom is caused by this styling:
Screen Shot 2021-04-13 at 3 03 05 PM

But removing it cause other styling issue so i kept it for now.

04/21 updated changes:
Screen Shot 2021-04-21 at 4 41 09 PM

### CODE REVIEW

File: ui/src/index.scss [hunk 1]
```diff
@@ -4,8 +4,28 @@
...
 
 :root {
-  --mediumblue: #e5eefd;
+
+  /* Factoring app coloring */
+  --lightBlue: #f4f7ff;
+  --button: #4874d2;
+  --navBar: #46496c;
+  --content-background-color: #474a6f;
+  --main-background-color: #171b44;
```
Comment (alexmatson-da) [inline ui/src/index.scss:13]: Better to stick with the "kebab-case" naming style for consistency. So using `--light-blue` and `--nav-bar` instead!

**Comment by alexmatson-da** [comment]
Looks good to me, the only concern I have is that the contrast for the secondary buttons ("Submit" and "Login") is very low against the background, and hard to see. 

It would be a great improvement to make those borders + inner text white

File: ui/src/index.scss (hunk 1 above)
Comment (JudithBLinder) [reply ui/src/index.scss:13]: ok :) I'll change it.

**Review by Sarahb-da — APPROVED** [review]

**Review by alexgraham-da — APPROVED** [review]
Looks good, thanks for this! I definitely agree with @alexmatson-da about the buttons though.

**Review by alexmatson-da — APPROVED** [review]
looks great, thank you!

**Pull request merged**